docs/source/operations/guide-use-armada.rst

```diff
@@ -62,9 +68,13 @@ Usage
 
     docker exec -it armada armada apply /examples/openstack-helm.yaml [ --debug-logging ]
 
-5. To upgrade charts just modify the armada yaml or chart code and re-run ``armada
+5. Upgrading charts: modify the armada yaml or chart source code and run ``armada
    apply`` above
 
+.. code:: bash
+
+    docker exec -it armada armada apply /examples/openstack-helm.yaml [ --debug-logging ]
```

Comment:
NIT: why do you open an interactive session `-it` for each of these commands? is it required?